index.js

```diff
@@ -88,27 +104,54 @@ class CSVPlugin {
     return this.context.json.expand(data)
   }
 
+  loadExportTemplates(data) {
+    const state = this.context.window.store?.getState()
+    const inferredItemTemplate = data['@graph']?.[0]?.template
+    const itemTemplate = loadTemplate(
+      state,
+      this.options.itemTemplate,
+      inferredItemTemplate)
+
+    if (!itemTemplate)
+      // eslint-disable-next-line max-len
+      throw new Error(`Failed to find specified item template "${this.options.itemTemplate}" or fallback "${data['@graph']?.[0]?.template}". Please install one of these templates, or select a different template and try again.`)
```

Comment:
Because we allow longer lines fro string literals you can also trick the linter like this:
```js
throw new Error(
   'long string ...'
)
```
Then we don't need the linter comment. For if statements without braces and a single expression, I'd generally make sure that the expression is really on the next line after the `if`. So I'd either remove the comment (or move it to the end of the line with the `if` statement), or just use braces for the if-branch just in case. I like if's without braces for stylistic reasons but they are little dangerous in JavaScript.